src/Item_Rack.php

```diff
@@ -747,9 +751,9 @@ private static function getCell($cell, $readonly = false)
                          ? $item->fields['otherserial']
                          : "";
             $model       = is_object($item)
-                        && is_object($item->model)
-                        && isset($item->model->fields['name'])
-                         ? $item->model->fields['name']
+                        && is_object($cell['model'])
```

Comment:
this change seems strange. Are you sure to move item to cell variable ?

Reply:
`$cell` is the parameter that will receive `$outbound` or `$data` items defined here:
![image](https://user-images.githubusercontent.com/33253653/158386168-2c8e96bb-807e-4f30-9148-4590250029a8.png)
![image](https://user-images.githubusercontent.com/33253653/158386191-8df0d787-dd97-42bc-bc72-c3953e53f610.png)